spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/CannotDeleteNonParentTaskExecutionException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.repository;
+
+import java.util.Set;
+
+import org.springframework.cloud.task.repository.TaskExecution;
+import org.springframework.util.StringUtils;
+
+/**
+ * @author Gunnar Hillert
+ */
+public class CannotDeleteNonParentTaskExecutionException extends RuntimeException {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
Following what @jvalkeal suggested remove the empty comment block and do a generated version instead of just 1L

Reply:
Fixed.